website/versioned_sidebars/version-5.x-sidebars.json

```diff
@@ -88,6 +88,16 @@
         }
       ]
     },
+    {
+      "type": "category",
+      "label": "Concept: Single spa modules",
```

Comment:
Honestly, I think this should go under the "Concept: Microfrontends" label, instead of a whole new one. To me this is just an extension of the microfrontend concept.